### PR TITLE
Added the app content into Vite project

### PR DESCRIPTION
**Pull request by RehanPaismo**

### CODE REVIEW

File: src/Components/AddToDo.jsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import React, { useState } from "react";
+import "./styles.css";
+
+const AddToDo = () => {
+  const [list, setList] = useState([]);
+  const [todo, setTodo] = useState();
+
+  const Add = () => {
+    if (todo != ""){
```
Comment (saqlain-paismo) [inline src/Components/AddToDo.jsx]: always use `===` or in case of negation values use `!==`

File: src/Components/AddToDo.jsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    <div className="container">
+      <h1>TODO APP</h1>
+      <div className="inputDiv">
+        <span>
+          <input
+            type="text"
+            value={todo}
+            onChange={(e) => {
+              setTodo(e.target.value);
+            }}
```
Comment (saqlain-paismo) [inline src/Components/AddToDo.jsx]: always try to create separate functions instead of writing in line

File: src/Components/AddToDo.jsx [hunk 3]
```diff
@@ -0,0 +1,41 @@
+import React, { useState } from "react";
+import "./styles.css";
+
+const AddToDo = () => {
+  const [list, setList] = useState([]);
+  const [todo, setTodo] = useState();
+
+  const Add = () => {
```
Comment (saqlain-paismo) [inline src/Components/AddToDo.jsx]: function and variable names always starts with small case

File: src/Components/TodoClass.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
+class ToDoClass {
```
Comment (saqlain-paismo) [inline src/Components/TodoClass.js:1]: why do we have a component with todoClass name?

**Review by saqlain-paismo — CHANGES_REQUESTED** [review]

File: src/Components/TodoClass.js (hunk 4 above)
Comment (RehanPaismo) [reply src/Components/TodoClass.js:1]: as i started implementing MobX, so there this component is here.

File: src/Components/AddToDo.jsx (hunk 3 above)
Comment (RehanPaismo) [reply src/Components/AddToDo.jsx]: Ok, Understand

File: src/Components/AddToDo.jsx (hunk 1 above)
Comment (RehanPaismo) [reply src/Components/AddToDo.jsx]: Understand.

File: src/Components/AddToDo.jsx (hunk 2 above)
Comment (RehanPaismo) [reply src/Components/AddToDo.jsx]: Understand

**Review by saqlain-paismo — APPROVED** [review]

**Pull request merged**